Source/WTF/wtf/PlatformHave.h

```diff
@@ -803,7 +803,7 @@
 #endif
 #endif
 
-#if PLATFORM(IOS) || PLATFORM(MAC) || PLATFORM(VISION) || PLATFORM(APPLETV)
+#if PLATFORM(IOS) || PLATFORM(MAC) || PLATFORM(MACCATALYST) || PLATFORM(VISION) || PLATFORM(APPLETV)
```

Comment:
I wonder if this would be best to negate (i.e. everything but watchOS). Not sure if we have a best practice for enumerating all platforms vs. excluding some.